api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/EriDSDCohortQueries.java

```diff
@@ -447,8 +446,18 @@ public CohortDefinition getPatientsWithViralLoadLessThan1000Within12Months() {
             + "                OR "
             + "                (o.concept_id=${hivViralLoadQualitative} AND o.value_coded IN (${beyondDetectableLimit},${undetectableViralLoad},${lessThan10Copies},${lessThan20Copies},${lessThan40Copies},${lessThan400Copies},${lessThan839Copies})) "
             + "            )  "
-            + "        AND e.location_id= :location  "
-       
+            + "        AND e.location_id= :location "
```

Comment:
Are these changes related to the work outline in this ticket? this looks like DSD report

Reply:
it might be because of the  code format  `mvn clean install`.